### ISSUE TITLE
Loss result inGptNeoForCasual

### ISSUE THREAD

**Issue by Yossillamm** [huggingface/datasets#2278]
Is there any way you give the " loss" and "logits" results in the gpt neo api?

**Comment by lhoestq** [comment]
Hi ! I think you might have to ask on the `transformers` repo on or the forum at https://discuss.huggingface.co/

Closing since it's not related to this library
